admin/template/template.php

```diff
@@ -2,12 +2,18 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<?php echo template_header(true); ?>
+	<?php echo template_header(true);
+	$title_full =  (isset($title) ? $title . $config['title_separator'] : '') . $config['lua']['serverName'];
+	?>
+
+	<title><?php echo $title_full ?></title>
+	<link rel="shortcut icon" href="<?php echo BASE_URL; ?>images/favicon.ico" type="image/x-icon" />
```

Comment:
Hello,

This favicon.ico file seems to not exist. Did you found any appropriate icon?

Reply:
Hi Slawkens,

I included the code just so people could add their own if needed and it would display on the page when bookmarked.
I was just using a random icon for my server.

This part can be removed if not needed.

Reply:
ok, thanks for the reply. I will merge it now.